build-apisix-base.sh

```diff
@@ -4,6 +4,11 @@ set -x
 
 version=${version:-0.0.0}
 
+OPENRESTY_VERSION=${OPENRESTY_VERSION:-1.21.4.1}
```

Comment:
What is the difference between OPENRESTY_VERSION and version？

Reply:
It is the version of apisix-base.